unittests/test_autodetect.py

```diff
@@ -55,13 +55,23 @@ def default_exec_ctx(make_exec_ctx_g, temp_topo):
 
 
 @pytest.fixture
-def remote_exec_ctx(make_exec_ctx, temp_topo):
+def remote_exec_ctx(request, make_exec_ctx, temp_topo):
     if test_util.USER_CONFIG_FILE is None:
         pytest.skip('no user configuration file supplied')
 
-    ctx = make_exec_ctx(test_util.USER_CONFIG_FILE,
-                        test_util.USER_SYSTEM,
-                        {'general/remote_detect': True})
+    custom_install = getattr(request, 'param', False)
+
+    if not custom_install:
+        ctx = make_exec_ctx(test_util.USER_CONFIG_FILE,
+                            test_util.USER_SYSTEM,
+                            {'general/remote_detect': True})
+    else:
+        ctx = make_exec_ctx(test_util.USER_CONFIG_FILE,
+                            test_util.USER_SYSTEM,
+                            {'general/remote_detect': True,
+                             'general/remote_install': [
+                                 'echo \'{"dummy": "value"}\' > topo.json',
```

Comment:
I don't understand the purpose of this. Maybe it's a leftover from the previous implementation. I've proposed below a simple `echo`.